apps/preview/next/pages/showcases/Footer/FooterBasic.tsx

```diff
@@ -0,0 +1,267 @@
+  },
+  {
+    label: 'Instagram',
+    link: '/instagram',
+    icon: 'SfIconInstagram',
+  },
+  {
+    label: 'Pinterest',
+    link: '/pinterest',
+    icon: 'SfIconPinterest',
+  },
+  {
+    label: 'Youtube',
+    link: '/youtube',
+    icon: 'SfIconYoutube',
+  },
+];
+const contactOptions = [
+  {
+    label: 'Help center',
+    link: '/help',
+    details: ['Find answers online anytime'],
+    icon: 'SfIconHelp',
+  },
+  {
+    label: 'Live chat',
+    link: '/live',
+    details: ['Mon–Fri, 5am–10pm PT', 'Sat–Sun, 6am–9pm PT'],
+    icon: 'SfIconContactSupport',
+  },
+  {
+    label: '1 234 567 8901',
+    link: '/tel',
+    details: ['Mon–Fri, 5am–10pm PT', 'Sat–Sun, 6am–9pm PT'],
+    icon: 'SfIconCall',
+  },
+];
+const bottomLinks = [
+  {
+    label: 'Terms',
+    link: '/terms',
+  },
+  {
+    label: 'Privacy policy',
+    link: '/privacy',
+  },
+];
+
+const iconsMap: {
+  [key: string]: any;
+} = {
+  SfIconFacebook,
+  SfIconTwitter,
+  SfIconInstagram,
+  SfIconPinterest,
+  SfIconYoutube,
+  SfIconHelp,
+  SfIconContactSupport,
+  SfIconCall,
+};
+export default function FooterBasic() {
+  return (
+    <footer className="pt-10 bg-neutral-100">
+      <div className="grid justify-center grid-cols-[1fr_1fr] md:grid-cols-[repeat(4,1fr)] px-4 md:px-6 pb-10 max-w-[1536px] mx-auto">
+        {categories.map(({ label, subcategories }) => (
+          <div className="grid grid-cols xs:pb-4" key={label}>
+            <div className="ml-4 text-lg font-medium leading-7 text-neutral-900 font-body">{label}</div>
+            {subcategories?.map(({ subcategoryLabel, link }) => (
+              <SfListItem className="py-2 !bg-transparent typography-text-sm font-body" key={subcategoryLabel}>
+                <SfLink
+                  className="no-underline text-neutral-600 hover:underline hover:!text-neutral-900 active:underline active:!text-neutral-900"
+                  variant="secondary"
+                  href={link}
+                >
+                  {subcategoryLabel}
+                </SfLink>
+              </SfListItem>
+            ))}
+          </div>
+        ))}
+      </div>
+      <hr />
+      <div className="py-10 md:flex md:mx-auto max-w-[1536px]">
+        {contactOptions.map(({ label, icon, link, details }) => {
+          const Icon = iconsMap[icon];
+          return (
+            <div className="mx-auto my-4 text-center" key={label}>
+              <Icon size="lg" />
+              <p className="py-1 my-2 font-medium typography-text-lg font-body">
+                <SfLink
+                  variant="secondary"
+                  className="no-underline text-neutral-600 hover:underline hover:!text-neutral-900 active:underline active:!text-neutral-900"
+                  href={link}
+                >
+                  {label}
+                </SfLink>
+              </p>
+              {details?.map((option) => (
+                <p className="text-sm leading-5 text-neutral-600 font-body" key={option}>
```

Comment:
There is `text-sm` class instead of `typography-text-sm`. I think we should always use our typography classes.

Reply:
changed thx